### ISSUE TITLE
Animated Headline element after updating to Elementor Pro Version 2.8.4

### ISSUE THREAD

**Issue by efintina** [elementor/elementor#10585]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description
After updating Elementor Pro to Version 2.8.4, I have an issues with Animated Headline elements. Actually the rotating text is not displayed in correct position. You can see the difference into the files that was attached.
[before-update.pdf](https://github.com/elementor/elementor/files/4215438/before-update.pdf)
[after-update.pdf](https://github.com/elementor/elementor/files/4215439/after-update.pdf)

## Steps to reproduce
Use Animated Headline element. Update Elementor Pro from previous version to  v2.8.4

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: WINNT
	Software: Apache/2.4.41 (Win64) PHP/7.3.12
	MySQL version: mariadb.org binary distribution v10.4.10
	PHP Version: 7.3.12
	PHP Max Input Vars: 2500
	PHP Max Post Size: 8M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.3.2
	Site URL: http://localhost/avantaje-publisind
	Home URL: http://localhost/avantaje-publisind
	WP Multisite: No
	Max Upload Size: 2 MB
	Memory limit: 256M
	Permalink Structure: /%category%/%postname%/
	Language: en-US
	Timezone: 2
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.1
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.2.2
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.106 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields
		Version: 5.8.7
		Author: Elliot Condon

	Akeeba Backup Professional for WordPress
		Version: 7.0.2
		Author: Akeeba Ltd

	Autoptimize
		Version: 2.6.1
		Author: Frank Goossens (futtta)

	Classic Editor
		Version: 1.5
		Author: WordPress Contributors

	DynamicConditions
		Version: 1.4.4
		Author: RTO GmbH

	Elementor
		Version: 2.9.2
		Author: Elementor.com

	Elementor Pro
		Version: 2.8.4
		Author: Elementor.com

	GDPR Cookie Consent
		Version: 2.2.4
		Author: WebToffee

	Premium Addons for Elementor
		Version: 3.11.2
		Author: Leap13

	SEOPress
		Version: 3.8.1
		Author: SEOPress

	ShortPixel Image Optimizer
		Version: 4.16.1
		Author: ShortPixel

	Site Kit by Google
		Version: 1.3.1
		Author: Google

	The Plus Addons for Elementor Page Builder Lite
		Version: 1.1.2
		Author: POSIMYTH Themes

	TinyMCE Advanced
		Version: 5.3.0
		Author: Andrew Ozz


== Must-Use Plugins ==
	akeeba-backup-coreupdate.php
		Version: 
		Author: 


== Elements Usage ==
	
	archive : 2
		archive-posts : 2
		heading : 1
		search-form : 1
		text-editor : 1
		theme-archive-title : 1
	footer : 1
		button : 1
		html : 1
	header : 1
		nav-menu : 1
		premium-addon-progressbar : 1
		theme-site-logo : 1
		theme-site-title : 1
	popup : 1
		button : 1
		countdown : 1
		heading : 2
		image : 1
	single : 2
		button : 1
		heading : 4
		image : 3
		post-navigation : 1
		premium-addon-button : 3
		text-editor : 4
		theme-post-content : 1
		theme-post-featured-image : 1
		theme-post-title : 1
	wp-post : 2
		text-editor : 2
		tp-accordion : 2
	wp-page : 5
		animated-headline : 2
		flip-box : 6
		form : 1
		heading : 1
		image : 1
		login : 1
		text-editor : 5
		theme-page-title : 1


== Log ==
	
Log: showing 10 of 102020-01-03 15:49:50 [info] elementor::elementor_updater Started 
2020-01-03 15:49:50 [info] Elementor/Upgrades - _v_2_8_3_recalc_usage_data Start  
2020-01-03 15:49:51 [info] Elementor/Upgrades - _v_2_8_3_recalc_usage_data Finished 
2020-01-03 15:49:51 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.8.2',
  'to' => '2.8.3',
)]
2020-01-03 15:49:52 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '2.8.2',
  'to' => '2.8.3',
)]
2020-01-03 15:50:46 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.8.2',
  'to' => '2.8.3',
)]
2020-01-19 21:01:14 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.8.3',
  'to' => '2.8.4',
)]
2020-01-28 09:35:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.8.4',
  'to' => '2.8.5',
)]
2020-02-17 19:47:08 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.8.5',
  'to' => '2.9.2',
)]
2020-02-17 19:56:50 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.8.3',
  'to' => '2.8.4',
)]

JS: showing 1 of 1JS: 2020-02-17 12:08:00 [error X 1][http://avantaje-publisind/wp-content/plugins/elementor/assets/js/frontend.min.js:2:90799] Cannot read property \'getCurrent\' of undefined 


				
```
</details>

**Comment by kingneb** [comment]
I am having the exact same problem. I have disabled all plugins but Elementor and Elementor Pro. Still an issue. The rotating text will only show up after the "Before text" if i make the font small enough to fit on a single line. 

<img width="1111" alt="Screen Shot 2020-02-17 at 4 41 42 PM" src="https://user-images.githubusercontent.com/61166615/74688196-c6cc8500-51a4-11ea-829d-ed5133f1a801.png">

**Comment by steve231293** [comment]
I have same problem,
You can use this custom css to make it working in Elementor 2.8.4 until Elementor team fix it.

.```
elementor-widget-animated-headline {
	.elementor-headline {
		display: block;
	}
}

.elementor-headline .elementor-headline-plain-text:first-of-type:after,
.elementor-headline .elementor-headline-plain-text:last-of-type:before {
	display: none !important;
}

.elementor-headline--style-rotate .elementor-headline-dynamic-wrapper {
	overflow: visible !important;
}
```

**Comment by anatolytkach** [comment]
Hi Steve, have applied this, but the code produces errors:
https://monosnap.com/direct/XNs2ro21atMaQXfy2lqlxoTXMt7n5Q
Any idea?

**Comment by steve231293** [comment]
@anatolytkach My code is sass then please try using this code: 

```
.elementor-widget-animated-headline 	.elementor-headline {
  display: block;
}

.elementor-headline .elementor-headline-plain-text:first-of-type:after,
.elementor-headline .elementor-headline-plain-text:last-of-type:before {
	display: none !important;
}

.elementor-headline--style-rotate .elementor-headline-dynamic-wrapper {
	overflow: visible !important;
}
```

**Comment by efintina** [comment]
Hello Steve,
Thank you very much for your kindness in sharing your solutions with us.
Your code worked for me with a few minor changes.

.elementor-widget-animated-headline .elementor-headline {
  display: block;
  text-align: center;
}

.elementor-headline .elementor-headline-plain-text:first-of-type:after,
.elementor-headline .elementor-headline-plain-text:last-of-type:before {
	display: none;
}

.elementor-headline--style-rotate .elementor-headline-dynamic-wrapper {
	overflow: visible;
}

**Comment by anatolytkach** [comment]
Steve, thanks a lot for your help and effort — the last one has worked as a charm!

**Comment by GOvEy1nw** [comment]
@efintina Please can you re-open this issue because it is still an actual issue that the Elementor team needs to fix and I wouldn't want them to miss it because this is marked as closed.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v2.9.1

Feel free to update

Thanks!

**Comment by jsuch11** [comment]
Hi, the update correctly fixes the live site, but not when editing the page in Elementor page builder..
